### PR TITLE
Service Image Seeds

### PR DESCRIPTION
**Pull request by Fire-Bringer**
Coded a number of edits to the service controller, index view and seeds to finish 12 types:

image

### CODE REVIEW

File: app/models/service.rb [hunk 1]
```diff
@@ -1,7 +1,8 @@
 class Service < ApplicationRecord
   belongs_to :user
   has_many :bookings, dependent: :destroy
-
+  has_one_attached :photo
```
Comment (HappyDay101) [inline app/models/service.rb:4]: Im thinking once we build everything, we can add to the backlog to change this to has many so we can have carousel images in the service index

File: db/seeds.rb
Comment (HappyDay101) [inline db/seeds.rb:1]: Thank you for doing the hard work and writing it many times for different images! We can come up with ideas when everything is build on how to refactor together!

**Review by HappyDay101 — APPROVED** [review]
Looks good, thank you!

File: app/models/service.rb (hunk 1 above)
Comment (Fire-Bringer) [reply app/models/service.rb:4]: Good point. That's exactly what I'm doing now for the current Service Show view now.

**Pull request merged**